src/pedalintandem.py

```diff
@@ -0,0 +1,212 @@
+import json
+from datetime import datetime, timedelta
+from curl_cffi import requests
+from common.tz import IST
+from bs4 import BeautifulSoup
+import re
+
+BASE_URL = "https://www.pedalintandem.com"
+
+def fetch_events_links(session):
+	res = session.get(f"{BASE_URL}/experiences")
+	soup = BeautifulSoup(res.text, 'html.parser')
+	events = soup.select('div.single-experience')
+
+	event_links = map(lambda x: x.find('a')['href'], events)
+
+	return event_links
+
+def fetch_events(event_links, session):
+	events = []
+	for event_link in event_links:
+
+		event_page = session.get(f"{BASE_URL}{event_link}")
+		event = BeautifulSoup(event_page.text, 'html.parser')
+		location = event.select_one('div.location').get_text().strip()
+
+		date_selector = event.select_one('div.product-variations-varieties select')
+
+		# Fetch data of the events which would be happening in future otherwise skip
+		if ( "disabled" in date_selector.attrs):
+			continue
+		
+		if not ( bool(re.search('bangalore', location, re.IGNORECASE)) or
+			bool(re.search('bengaluru', location, re.IGNORECASE)) or
+			bool(re.search('arkavathi', location, re.IGNORECASE)) or
+			bool(re.search('avalahalli', location, re.IGNORECASE)) or
+			bool(re.search('avathi', location, re.IGNORECASE)) or
+			bool(re.search('devarayanadurga', location, re.IGNORECASE)) or
+			bool(re.search('gunjur', location, re.IGNORECASE)) or
+			bool(re.search('hennur', location, re.IGNORECASE)) or
+			bool(re.search('Hesaraghatta', location, re.IGNORECASE)) or
+			bool(re.search('kanakapura', location, re.IGNORECASE)) or
+			bool(re.search('malleshwaram', location, re.IGNORECASE)) or
+			bool(re.search('indiranagar', location, re.IGNORECASE)) or
+			bool(re.search('manchanabele', location, re.IGNORECASE)) or
+			bool(re.search('pedal', location, re.IGNORECASE)) or
+			bool(re.search('pitstop', location, re.IGNORECASE)) or 
+			bool(re.search('rajankunte', location, re.IGNORECASE)) ):
+			continue
+
+		duration = event.select_one('div.duration').get_text().strip()
+		if bool(re.search('/', duration)):
+			continue
+
+		events.append(event)
+
+	return events
+
+def make_event(event):
+	heading = event.select_one('div.heading').get_text().strip()
```

Comment:
I think bs4 recommends using  `.text` instead. Check once

Reply:
In the official documentation, it is only using `get_text()` and there is no mention of `.text`. But on other places on internet mentioned that using `.text` is slightly preferred but they are similar. 
I also tried them both, the output is same so it can be used.
Should I use `.text` or `get_text()`